tests/test.c

```diff
@@ -647,3 +647,28 @@ void check_param_range(char* dev_name, uint64_t uid, char* param_name, param_typ
     }
     print_pass();
 }
+
+// Returns if the latency is neither too high nor too low
+void check_latency(char* dev_name, uint64_t uid, char* param_name, int32_t lower_bound_latency, int32_t upper_bound_latency, int32_t start_time) {
```

Comment:
I think it's probably fine to hard code the "lower bound" as 0 here. We shouldn't punish our system for being too fast! Though checking that it's at least 0 is a validity check because negative latency means something is wrong.